internal/app/rds/rds_test.go

```diff
@@ -251,6 +251,8 @@ func TestGetDBInstanceStatusCode(t *testing.T) {
 		{input: "available", want: rds.InstanceStatusAvailable},
 		{input: "stopped", want: rds.InstanceStatusStopped},
 		{input: "backing-up", want: rds.InstanceStatusBackingUp},
+		{input: "creating", want: rds.InstanceStatusCreating},
+		{input: "deleting", want: rds.InstanceStatusDeleting},
```

Comment:
Should we also add a test for `InstanceStatusStopping` ?

Reply:
Good catch, I added it to the tests